CDP4IMEInstaller/CDP4IMEInstaller.wixproj

```diff
@@ -97,16 +104,15 @@
     <Content Include="ThirdPartyLicense\Hardcodet.Wpf.TaskbarNotification.txt" />
     <Content Include="ThirdPartyLicense\IronPython.txt" />
     <Content Include="ThirdPartyLicense\Markdown.Xaml.txt" />
+    <Content Include="ThirdPartyLicense\Microsoft.AspNet.Mvc.txt" />
+    <Content Include="ThirdPartyLicense\Microsoft.AspNet.WebApi.Client.txt" />
+    <Content Include="ThirdPartyLicense\Microsoft.CSharp.txt" />
+    <Content Include="ThirdPartyLicense\Microsoft.NET.Test.Sdk.txt" />

```

Comment:
I think test Sdk and Moq should not be added here, because we do not deliver test stuff in the Installer.